app/views/projects/_edit_form.html.erb

```diff
@@ -95,120 +94,217 @@
-            <div class="custom_error" id="directory_error" aria-live="polite">
-               <%= image_tag("custom_error.svg", alt: "This field is required") %>
-               <div class="error_message">This field is required. </div>
+            <select id="departments" name="departments[]"  aria-label="project department" required value class="form-select" multiple >
+                           <% Affiliation.all.each do |affiliation| %>
+                           <% if @project.departments.include?(affiliation[:code]) %>
+                           <option selected><%= affiliation[:code] %></option>
+                           <% else %>
+                           <option><%= affiliation[:code] %></option>
+                           <% end %>
+                           <% end %>
+                        </select>
             </div>
          </div>
-       </div>
+      </div> <!-- row -->
 
+      <div class="row ">
+         <div class="col-md-3">
+            <label for="description">Description</label>
+         </div>
+         <div class="col-md-9">
+            <textarea type="text" id="description"  aria-label="project description" name="description" class="input-text-long"
+                        rows="5" cols="72" placeholder=""><%= @project.metadata_model.description %></textarea>
+         </div>
+      </div> <!-- row -->
 
-       <div class="row ">
-  <div class="col-md-3">
+      <!-- Keeps track of added read-only and read-write users -->
+      <input type="text" id="ro_user_counter" name="ro_user_counter" value="0" style="display:none;"/>
+      <input type="text" id="rw_user_counter" name="rw_user_counter" value="0" style="display:none;"/>
 
-  <label for="departments">Departments</label>
-  <div class="required-field">Required</div>
-  </div>
-  <div class="col-md-9">
+      <div class="actions">
+         <button type=submit value="Submit" class="btn btn-primary" id="submit-btn">Submit</button>
+         <% if params[:action] == 'edit' %>
+         <%= link_to "Cancel", @project, class: "btn btn-secondary", id: "cancel-btn" %>
+         <% else %>
+         <%= link_to "Cancel", root_path, class: "btn btn-secondary", id: "cancel-btn" %>
+         <% end %>
+      </div>
 
-  <select id="departments" name="departments[]"  aria-label="project department" required value class="form-select" multiple >
-               <% Affiliation.all.each do |affiliation| %>
-               <% if @project.departments.include?(affiliation[:code]) %>
-               <option selected><%= affiliation[:code] %></option>
-               <% else %>
-               <option><%= affiliation[:code] %></option>
-               <% end %>
-                <% end %>
-            </select>
-  </div>
-</div>
-</div>
+</div> <!-- id=edit -->
+
+<script>
```

Comment:
This is the biggest change in this PR: the JavaScript logic to get the autocomplete and the validation to work together.